src/Controller/Api/MailArchiveController.php

```diff
@@ -90,9 +97,6 @@ public function download(Request $request): JsonResponse
             throw MailArchiveException::notFound();
         }
 
-        // for backward compatibility
-        $content = $mailArchive->getEml();
```

Comment:
create $content as empty variable or check for eml path and throw an error when it is not found 🤔. Feels weird to define $content only in a if and use it outside of the if